src/common/utility.c

```diff
@@ -671,16 +671,8 @@ gchar *dt_util_normalize_path(const gchar *_input)
     char *current_dir = g_get_current_dir();
     char *tmp_filename = g_build_filename(current_dir, filename, NULL);
     g_free(filename);
-    filename = g_realpath(tmp_filename);
-    if(filename == NULL)
-    {
-      g_free(current_dir);
-      g_free(tmp_filename);
-      g_free(filename);
-      return NULL;
-    }
+    filename = tmp_filename;
```

Comment:
Now you set filename to the full path name without resolving the symlinks.

This seems quite wrong to me, and at least does not correspond to the PR subject above.

If I'm confused, please explain. TIA.

Reply:
The other code path, in which darktable got a "full path name", doesn't resolve symlinks, too.

Which is the inconsistency the OP in the original PR talked about.